src/scripts/popup.js

```diff
@@ -12,39 +12,44 @@ import { RandomUserGenerator } from './generators/RandomUserGenerator';
 import { UiNamesGenerator } from './generators/UiNamesGenerator';
 import { RandomProfileGenerator } from './generators/RandomProfileGenerator';
 
-$(document).ready(function () {
+$(function () {
     var nameGenerator = new NameGeneratorExtension(document, [
         new FakerGenerator(),
         new RandomUserGenerator(),
         new UiNamesGenerator(),
         new RandomProfileGenerator()
     ]);
 
-    $('#refreshbtn').click(function () {
+    $('#refreshbtn').on("click", function () {
```

Comment:
Are there any benefits of using this type of subscription?

Reply:
Since JQuery 3.+ these methods are depricated [link](https://github.com/jquery/jquery-migrate/blob/main/warnings.md#jqmigrate-jqueryfnclick-event-shorthand-is-deprecated) so I replaced them.